common/options.go

```diff
@@ -0,0 +1,58 @@
+package common
+
+import (
+	"fmt"
+	"reflect"
+	"strconv"
+	"time"
+
+	"github.com/dop251/goja"
+)
+
+func parseBoolOpt(key string, val goja.Value) (b bool, err error) {
+	if b, err = strconv.ParseBool(val.String()); err != nil {
+		return false, fmt.Errorf("%s should be a boolean: %w", key, err)
+	}
+	return
+}
+
+func parseStrOpt(key string, val goja.Value) (s string, err error) {
+	if val.ExportType().Kind() != reflect.String {
+		return "", fmt.Errorf("%s should be a string", key)
+	}
+	return val.String(), nil
```

Comment:
Right, I'm going to handle `null` cases next. Please see the description:

> I happened to fix-ed https://github.com/grafana/xk6-browser/issues/407 while I was working on this PR. Ankur (since https://github.com/grafana/xk6-browser/issues/407 was assigned to you), I can post the fix in a different PR if you don't mind.

We will override the defaults and return errors if we have `nil` checks here. I'll explain the reason in detail in the next PR.

Reply:
Sounds good, but let's not merge this PR until that is fixed, since panicking on `null` is a regression from `main`. So that other PR can be merged here.

Reply:
Please see #616.